### ISSUE TITLE
Merit-Capped Voting

### ISSUE THREAD

**Issue by yeqbfgxjiq** [aragoncoop/ideas#4]
Like [karma-capped voting](https://www.reddit.com/r/daonuts/comments/ao5umv/why_weight_minrep_token_for_community_governance/), but a little different. 2 tokens are minted: 1 as the base token, and another that weights the base token. Then whatever merit you acquire is in addition to your membership token, but there's a also a cap so whales don't dominate and everyone has a voice. This will (hopefully) create a meritocratic systems that is also somewhat sybil resistant  (depending on the membership process). Not perfect, but workable. Could also integrate quadratic voting with the merit token as well.

**Comment by lkngtn** [comment]
Is it assumed that the "base token" is non-transferrable, and "weight token" is transferrable?

**Comment by yeqbfgxjiq** [comment]
- MBR is a membership token which is determined by the membership rules
- MRT is a merit token which can be earned by earning bounties or being allocated merit by members

Trade-able merit tokens would be interesting, but for the coop I was thinking something more in line with redemption against a vault (Redemption App). Is that possible without making the tokens trade-able?

There's also the option to offer a liquidity pool on Uniswap, which would be similar to the Redemption App, but would eventually lead to tokens trading on open markets. Maybe that's a good thing though because it provides a way to get "instant access" to the community (like how on /r/ethereum you need 10 karma to comment, here you could buy the initial tokens to participate). That would only work if there's limited liquidity and/or a cap on merit weighting though.

**Comment by yeqbfgxjiq** [comment]
Oh also, speaking of membership rules. The current process is what it is, but if the Coop Workflow Proposal passes then it seems natural to use memership/merit voting to welcome new members on a rolling basis?
- one benefit is that you have the built in sybil resistance from day 1
- another is that it would scale as the community grows, for example people could do work and submit a proposal to get the bounty and become a member
- if someone needed to be voted in immediately (can't think of why, but if...) the coop can always organize to create "emergency" votes too

**Comment by lkngtn** [comment]
> Trade-able merit tokens would be interesting, but for the coop I was thinking something more in line with redemption against a vault (Redemption App). Is that possible without making the tokens trade-able?

I'm not sure, I think currently setting a token to non-transferrable means that it can really only be burned/minted from the token manager application. This is pretty rigid, and so transferrable tokens seem to be better suited for most use cases right now. For example you can't use a non-transferrable token with projects/bounties because it requires transferring the tokens to the bounties contract and then transferring them to a user. 

It would be nice if there was more a of a "conditional" non-transferrable token but not sure exactly how that would look.

**Comment by lkngtn** [comment]
> Oh also, speaking of membership rules. The current process is what it is, but if the Coop Workflow Proposal passes then it seems natural to use memership/merit voting to welcome new members on a rolling basis?

Totally, that has always been the intent, depending on volume of new members it may make sense to instead have a vote to issue membership tokens periodically and then have a number of individuals assigned to processing membership requests. Sort of what I proposed here: https://github.com/aragoncoop/ops/issues/1

**Comment by yeqbfgxjiq** [comment]
Yeah that would be great. The less overhead for the community the better so people can focus on building/doing. Added that to the [membership doc](https://github.com/aragoncoop/coop-dao-docs/blob/master/2_membership.md).
